execution_engine/src/core/engine_state/step.rs

```diff
@@ -39,13 +39,25 @@ impl RewardItem {
     }
 }
 
+#[derive(Debug)]
+pub struct DisableItem {
+    pub validator_id: PublicKey,
+}
+
+impl DisableItem {
+    pub fn new(validator_id: PublicKey) -> Self {
+        Self { validator_id }
+    }
+}
```

Comment:
If everything is `pub`, is this constructor necessary?

Reply:
Again, I'm only following the patterns from the existing code :stuck_out_tongue: 
I'm not sure what the rationale is here, I guess it's just a good coding practice to have a constructor.